modules/grafana-agent-loki/variables.tf

```diff
@@ -7,3 +7,8 @@ variable "loki_password" {
 variable "loki_hostname" {
   type = string
 }
+variable "k8s_namespace" {
+  type        = string
+  default     = "default"
+  description = "Kubernetes namespace to use"
```

Comment:
This is only for the DaemonSet / Deployments - the service accounts cluster role{s, bindings} are not namespaced.